### PR TITLE
Add simple time tracking sample

### PR DESCRIPTION
**Pull request by nielsenko**

### CODE REVIEW

File: time_track/bin/time_track.dart [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+@RealmModel()
+class _Category {
+  @PrimaryKey()
+  late final String name;
+}
+
+@RealmModel()
+class _Now {
+  late final int time; // no DateTime yet (stay tuned)
```
Comment (blagoev) [inline time_track/bin/time_track.dart:13]:
```suggestion
  late int time; // no DateTime yet (stay tuned)
```

File: time_track/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+```
+* Install native realm library (supported platforms are Windows, MacOS, and Linux).
+```
+    dart run realm_dart install
+```
+* Generate Dart binding for Realm Database
+```
+    dart run realm_dart generate
+```
+* Compile to native for speed
```
Comment (blagoev) [inline time_track/README.md:17]:
```suggestion
* Optional. Compile to native for speed
```

File: time_track/bin/time_track.dart [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      // Find latest time track entry
+      final latest = realm.query<Now>('TRUEPREDICATE SORT(time DESCENDING)').firstOrNull;
+
+      // If category changed, then create a new entry
+      if (latest?.category != category) {
+        final now = Now(
+          DateTime.now().millisecondsSinceEpoch,
+          category: category,
+        );
+        // Persist new entry
```
Comment (blagoev) [inline time_track/bin/time_track.dart:36]:
```suggestion
        // Add new entry
```

File: time_track/bin/time_track.dart (hunk 1 above)
Comment (blagoev) [reply time_track/bin/time_track.dart:13]: needs `generate`

**Review by blagoev — APPROVED** [review]
small suggestions

File: time_track/README.md (hunk 2 above)
Comment (nielsenko) [reply time_track/README.md]: Fixed

File: time_track/bin/time_track.dart (hunk 3 above)
Comment (nielsenko) [reply time_track/bin/time_track.dart]: Fixed

File: time_track/bin/time_track.dart (hunk 1 above)
Comment (nielsenko) [reply time_track/bin/time_track.dart]: Fixed

**Pull request merged**